From a1ae497ae3b37cc4ff0e6b1b4650e0ef7bc783f3 Mon Sep 17 00:00:00 2001
From: "Ranjan, Rajani" <rajani.ranjan@intel.com>
Date: Wed, 9 Aug 2023 12:05:52 +0530
Subject: [PATCH] changing sensors@aidl to version_2 in manifest.xml to inline
 with compatibilty matrix.

google compatibility matrix for android u requires sensors@aidl version 2.
android.hardware.sensors@2.0::ISensors/default is deprecated in
compatibility matrix at FCM Version 8.
The minimum requirement for Sensors HAL in android 14 should be AIDL
implementation to meet target-level 8, API level 34 requirement.

Tracked-On: OAM-111677
Signed-off-by: Ranjan, Rajani <rajani.ranjan@intel.com>
---
 sensors/aidl/sensors-aidl.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sensors/aidl/sensors-aidl.xml b/sensors/aidl/sensors-aidl.xml
index 7898a6b..36b28ed 100644
--- a/sensors/aidl/sensors-aidl.xml
+++ b/sensors/aidl/sensors-aidl.xml
@@ -1,7 +1,7 @@
 <manifest version="1.0" type="device">
     <hal format="aidl">
         <name>android.hardware.sensors</name>
-        <version>1</version>
+        <version>2</version>
         <fqname>ISensors/default</fqname>
     </hal>
 </manifest>
-- 
2.41.0

